src/Campaigns/Routes/RegisterCampaignRoutes.php

```diff
@@ -219,6 +219,10 @@ public function getSchema(): array
                     'description' => esc_html__('Campaign goal', 'give'),
                     'errorMessage' => esc_html__('Must be a number', 'give'),
                 ],
+                'goalProgress' => [
```

Comment:
Let's give this bad boy a description. 👍

Reply:
Resolved, see [dc26132](https://github.com/impress-org/givewp/pull/7628/commits/dc26132eafd187893a609103d78b83eb1e818ed6).